src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage`2.cs

```diff
@@ -152,5 +152,15 @@ protected override void Dispose(bool disposing)
         }
 
         protected abstract string RoslynLanguageName { get; }
+
+        private void EnableRemoteHostClientService()
+        {
+            ((RemoteHostClientServiceFactory.RemoteHostClientService)this.Workspace.Services.GetService<IRemoteHostClientService>()).Enable();
```

Comment:
it used to. I moved that into VS dll. and created new interface called IRemoteHostClientFactory.